### PR TITLE
Convert .tgs with better code

### PR DESCRIPTION
**Pull request by Benau**
As discussed in https://github.com/42wim/matterbridge/issues/1459 I made the PR, and it seems that it's not very difficult to keep the old code

Right now "making tgs smaller" is not yet done, but it should be easier to add after this PR is merged (and I think it's better to not  mix too many things in same PR)

### CODE REVIEW

**Comment by 42wim** [comment]
@Benau Thanks, looks good, could you make the linter happy and then we can merge it?

**Comment by Benau** [comment]
```go
	if err != nil {
 		return fmt.Errorf("Failed to run libtgsconverter.ImportFromData: %s", err.Error())
 	}

 	*data = blob
 	return nil
```
I guess this should fix the error now?

File: bridge/helper/libtgsconverter.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	return nil
+}
+
+// ConvertTgsToX convert input data (which should be tgs format) to any format supported by libtgsconverter
+func ConvertTgsToX(data *[]byte, outputFormat string, logger *logrus.Entry) error {
+	options := libtgsconverter.NewConverterOptions()
+	options.SetExtension(outputFormat)
+	blob, err := libtgsconverter.ImportFromData(*data, options)
+	if err != nil {
+		return fmt.Errorf("Failed to run libtgsconverter.ImportFromData: %s", err.Error())
```
Comment (42wim) [inline bridge/helper/libtgsconverter.go:21]:
```suggestion
		return fmt.Errorf("failed to run libtgsconverter.ImportFromData: %s", err.Error())
```

File: bridge/helper/libtgsconverter.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
+// +build cgo
+
+package helper
+
+import (
+	"fmt"
```
Comment (42wim) [inline bridge/helper/libtgsconverter.go:6]:
```suggestion
	"fmt"
	
```

File: bridge/helper/libtgsconverter.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
```
Comment (42wim) [inline bridge/helper/libtgsconverter.go:7]:
```suggestion

```

**Comment by 42wim** [comment]
@Benau thanks!

**Pull request merged**